doc/source/install/others.rst

```diff
@@ -242,6 +242,78 @@ files::
 
 Shows all ``configure`` options.
 
+``--enable-apache-arrow``
++++++++++++++++++++++++++
+
+Enables Apache Arrow features.
+
+The default is ``auto``. The features are enabled if Apache Arrow 
+is installed on the system, otherwise disabled.
+
+When you specify this parameter expressly, you should install Apache
+Arrow on the system beforehand.
+
+You can install Apache Arrow according to official installation procedure.
+Groonga requires either ``libarrow-dev`` or ``arrow-devel`` depending on the system.
+
+`Install Apache Arrow <https://arrow.apache.org/install/>`_
+
+Here is an example that enables Apache Arrow features expressly::
+
+  % ./configure --enable-apache-arrow
+
+.. note::
+
+   If you install Apache Arrow manually, use the :ref:`install-others-configure-pkg-config-path` option.
+
+``--disable-apache-arrow``
+++++++++++++++++++++++++++
+
+Disables Apache Arrow features.
+
+The default is ``false``.
+
+When this parameter is specified, Groonga disables Apache Arrow 
+features even if Apache Arrow is install on the system.
+
+Here is an example that disables Apache Arrow features::
+
+  % ./configure --disable-apache-arrow
+
+.. _install-others-configure-pkg-config-path:
+
+``PKG_CONFIG_PATH=PATHS``
++++++++++++++++++++++++++
+
+Adds search paths of ``.pc`` files for ``pkg-config``.
```

Comment:
Can you add an explanation that we can specify multiple paths by separating them with `:`?

Reply:
Sure, I will add it.

Reply:
Added.